code/mob/living/critter.dm

```diff
@@ -84,6 +84,9 @@ ABSTRACT_TYPE(/mob/living/critter)
 
 	var/pull_w_class = W_CLASS_SMALL
 
+	///Always call limb intent procs
+	var/intent_override = FALSE
```

Comment:
Since this is only being used for grabbing, would maybe be better renamed?

Also, what about moving to as a handHolder variable? Could maybe see wanting to have some limbs use grab but not others?